src/eredis_client.erl

```diff
@@ -231,10 +234,15 @@ handle_info({reconnect, _Reason}, State) ->
 handle_info(_Info, State) ->
     {noreply, State}.
 
-terminate(_Reason, #state{socket = undefined}) ->
+terminate(_Reason, #state{socket = undefined, sentinel = undefined}) ->
     ok;
+terminate(_Reason, #state{socket = Socket, transport = Transport, sentinel = undefined}) ->
+    Transport:close(Socket);
+terminate(_Reason, #state{socket = undefined}) ->
+    eredis_sentinel:stop();
```

Comment:
If the eredis_sentinel process is shared between many eredis_client processes, if it's stopped it affects the other eredis clients which are still running, right?